### PR TITLE
Feature aatf manage evidence note

### PR DESCRIPTION
**Pull request by NateHogan**

### CODE REVIEW

File: src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerCreateEvidenceNoteTests.cs [hunk 1]
```diff
@@ -224,6 +225,50 @@ public async Task CreateEvidenceNotePost_GivenModelIsValid_ShouldBeRedirectToVie
...
+            };
+
+            var result = await controller.Index(model) as RedirectToRouteResult;
+
+            result.RouteValues["action"].Should().Be("Index");
+            result.RouteValues["controller"].Should().Be("Holding"); // TODO Change to Create Evidence Note
+            result.RouteValues["organisationId"].Should().Be(model.OrganisationId);
+            //result.RouteValues["aatfId"].Should().Be(model.SelectedId); // This will be needed
+        }
+
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerCreateEvidenceNoteTests.cs:271]: Need test to ensure that the ManageEvidenceNoteViewModel is returned to the View

File: src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs [hunk 2]
```diff
@@ -28,6 +30,28 @@ public ManageEvidenceNotesController(IMapper mapper, BreadcrumbService breadcrum
...
+
+                await this.SetBreadcrumb(organisationId, BreadCrumbConstant.AatfManageEvidence);
+
+                return this.View(model);
+            }
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> Index(ManageEvidenceNoteViewModel model)
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs:50]: probably doesnt need to be async, just public ActionResult

File: src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs [hunk 3]
```diff
@@ -28,6 +30,28 @@ public ManageEvidenceNotesController(IMapper mapper, BreadcrumbService breadcrum
             this.apiClient = apiClient;
         }
 
+        [HttpGet]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs:33]: add unit test to check that the HttpGet and Post attributes are on the actions

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Index.cshtml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Core.Helpers
+@using EA.Weee.Web.RazorHelpers
+
+@model EA.Weee.Web.Areas.Aatf.ViewModels.ManageEvidenceNoteViewModel
+@{
+    var title = String.Format("Manage Evidence Note for {0}", Model.AatfName);
+    ViewBag.Title = title + " - GOV.UK";
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Index.cshtml:8]: Just have this as Manage evidence with the - GOV.UK part

**Review by JohnPadgettCivica — CHANGES_REQUESTED** [review]

**Review by JohnPadgettCivica — CHANGES_REQUESTED** [review]

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Index.cshtml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+}
+
+
+<section id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <p>
+                @Html.ActionLink("Back", "ChooseActivity", "Home", new { area = "Scheme", pcsId = Model.OrganisationId }, new { @class = "govuk-back-link" })
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Index.cshtml:17]: This should go back to Choose site

**Review by JohnPadgettCivica — CHANGES_REQUESTED** [review]

File: src/EA.Weee.Web/Constant/BreadCrumbConstant.cs [hunk 6]
```diff
@@ -3,6 +3,7 @@
     public static class BreadCrumbConstant
     {
         public static string AatfEvidence = "Manage AATF Evidence";
+        public static string AatfManageEvidence = "Manage AATF Evidence Notes";
         public static string AatfReturn = "Manage AATF returns";
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Constant/BreadCrumbConstant.cs:7]: I think the Manage AATF Evidence activity one should just be "Manage evidence notes" if you could update

Comment (GuidoWorkDev) [reply src/EA.Weee.Web/Constant/BreadCrumbConstant.cs:7]: wireframe says CCCI_00007 and content catalogue says "Manage evidence notes"

**Pull request merged**